index.html

```diff
@@ -7,92 +7,207 @@
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="forest.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Connection</header>
-          <p>We provide a good internet connection in our remotest cabins</p>
+
+<div class="whole_page_wrapper">
+
+
+  
+  <header class="header section flex_container">
+    <div class="logo">ReMotely</div>
+    <nav class="nav flex_container">
+      <div class="link_flex_container"><a class="links" href="#">book</a><a class="links" href="#">explore</a>
+        <a class="links" href="#">about</a>
+      </div>
+      <div class="sign_container"><a class="sign" href="#">sign in</a></div>
+    </nav>
+  </header>
+
+
+  <section class="section section_hero">
+    <div class="section_container">
+      <div class="column phrase">
+        <div class="catch_phrase">
+          <h1 class="invite_phrase">Work from <br> a cabin near Helsinki</h1>
+          <div class=button_container>
+            <button class="btn" type="submit">Explore</button>
+          </div>
         </div>
       </div>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="highway.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Accesibility</header>
-          <p>Every Cabin is up in the nature but also easy to get to and to the towns around</p>
+      <div class="column cottage">
+        <div class="photo">
+          <img class="main_photo" src="switzphoto.jpg" alt="chatka ze Szwajcarii">
         </div>
       </div>
     </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="mountains.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Close to nature</header>
-          <p>Every cabin means different areas to explore!</p>
+    
+  </section>
+
+ 
+  <section class="section second_page_container">
+    <h2 class="section_title">We offer a handpicked range of cabins for nature lovers in Europe to live and work
+      from for desired time.</h2>
+    <div class="all_pros_container">
+
+      <div class="pros_container">
+        <div class="photo_description_container">
+          <div class="pros_photo"><img class="pros_pic" src="forest.png" alt="zdjęcie zalet"></div>
+
+          <div class="header_container">
+            <h3 class="paragraph_header">Connection</h3>
+          </div>
+
+
+          <div class="paragraph_description">
+            <p class="text">We provide a good internet connection <br> in our remotest cabins</p>
```

Comment:
still not done ⛔

Reply:
the only idea I have in here is to create two separate paragraph in here... as a column but how is that better from simply break the line?